taccsite_cms/static/site_cms/css/src/_themes/export.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Export appropriate theme file based on settings value
+ */
+const rootPath = __dirname + '/../../../../../../';
+const settings = require( rootPath + 'taccsite_cms/settings.json');
```

Comment:
__Suggestion__
Consider storing path in settings. See [comment by @Strmiska on GH-101](https://github.com/TACC/Core-CMS/pull/222#discussion_r648736068).